sql/core/src/test/resources/sql-tests/results/join-lateral.sql.out

```diff
@@ -323,14 +322,10 @@ SELECT * FROM t1, LATERAL (SELECT rand(0) FROM t2)
 struct<>
 -- !query output
 org.apache.spark.sql.AnalysisException
-Non-deterministic lateral subqueries are not supported when joining with outer relations that produce more than one row
-SubqueryAlias __auto_generated_subquery_name
-+- Project [rand(0) AS rand(0)#x]
-   +- SubqueryAlias spark_catalog.default.t2
-      +- View (`spark_catalog`.`default`.`t2`, [c1#x,c2#x])
-         +- Project [cast(col1#x as int) AS c1#x, cast(col2#x as int) AS c2#x]
-            +- LocalRelation [col1#x, col2#x]
-; line 1 pos 9
```

Comment:
The problematic query plan will not get printed out after this change?

Reply:
This is a good question. Let's talk offline about it in general a bit.

Reply:
Update on this: it looks like @gengliangwang  has this PR to support adding the origin information to `AnalysisException`: https://github.com/apache/spark/pull/37841. We can think about whether to include or elide the logical plan strings in those cases.